### PR TITLE
docs: Add release notes for v1.8.0.

### PR DESCRIPTION
**Pull request by davecgh**

### CODE REVIEW

File: docs/release-notes/release-notes-1.8.0.md [hunk 1]
```diff
@@ -0,0 +1,851 @@
...
+* Miscellaneous network and protocol optimizations
+* Quality assurance changes
+
+For those unfamiliar with the
+[voting process](https://docs.decred.org/governance/consensus-rule-voting/overview/)
+in Decred, all code needed in order to support each of the aforementioned
+consensus changes is already included in this release, however it will remain
+dormant until the stakeholders vote to activate it.
+
+For reference, the consensus change work for was originally proposed and
```
Comment (jholdstock) [inline docs/release-notes/release-notes-1.8.0.md:36]:
```suggestion
For reference, the consensus change work was originally proposed and
```

File: docs/release-notes/release-notes-1.8.0.md [hunk 2]
```diff
@@ -0,0 +1,851 @@
...
+### Low Fee Transaction Rejection
+
+The default transaction acceptance and relay policy is no longer based on
+priority and instead now immediately rejects all transactions that do not pay
+the minimum required fee.
+
+This provides a better user experience for transactions that do not pay enough
+fees.
+
+For some insight into the motivation for this change, historically, prior to the
```
Comment (jholdstock) [inline docs/release-notes/release-notes-1.8.0.md:170]: Remove tautology

```suggestion
For some insight into the motivation for this change, prior to the
```

File: docs/release-notes/release-notes-1.8.0.md [hunk 3]
```diff
@@ -0,0 +1,851 @@
...
+[getwork JSON-RPC API Documentation](https://github.com/decred/dcrd/blob/master/docs/json_rpc_api.mediawiki#getwork)
+for API details.
+
+## Changelog
+
+This release consists of 439 git commits from 18 contributors which total to 408
+files changed, 25840 additional lines of code, and 22871 deleted lines of code.
+
+All commits since the last release may be viewed on GitHub
+[here](https://github.com/decred/dcrd/compare/release-v1.7.7...release-v1.8.0).
```
Comment (jholdstock) [inline docs/release-notes/release-notes-1.8.0.md:353]: This link says 14 contributors, but the text above says 18

**Review by jholdstock — COMMENTED** [review]
There is no mention of the new flag `--allowoldforks`

File: docs/release-notes/release-notes-1.8.0.md (hunk 1 above)
Comment (davecgh) [reply docs/release-notes/release-notes-1.8.0.md]: Updated.

**Comment by davecgh** [comment]
> There is no mention of the new flags `--allowoldforks` and `--boundaddrevents`

I debated whether or not to add that and decided against it because it's not something anyone who really doesn't know what they're doing should do, so I figured it wasn't worth calling attention to it.

**Review by jholdstock — APPROVED** [review]

**Review by jrick — APPROVED** [review]

**Pull request merged**